code/02-nix-commands/01-install-global-packages.md

```diff
@@ -1,5 +1,7 @@
 # Install Global Packages
 
+You can use Nix like a "traditional" package manager, and install packages "globally" (in the `$PATH` of all of your user's shells).
```

Comment:
We can introduce this in a more gentle way to non-Nix users. Perhaps something like:

> For newcomers, we can thought of Nix as a supercharged package manager. Similar to traditional package managers, we can use Nix to install packages globally. Nix installs a global package to `~/.nix-profile/bin`, which should be automatically included in your shell's `$PATH`.

Reply:
I feel like this is a little bit much technical detail, but I can't see what to cut from it. I'll add it as suggested but if anyone has a better idea drop a comment here.